pubspec.yaml

```diff
@@ -17,6 +17,12 @@ dependencies:
   dio: ^3.0.9
   uuid: ^2.0.4
 
+  #Sliding Up Panel
+  sliding_up_panel: ^1.0.0
```

Comment:
I don't see this dependency being imported by any of your changes. Is it just me being blind or is this being used somewhere? If not being used, we should remove the dependency from this PR, and only add it once required. Could be interesting for the popup info feature PR.

Reply:
Yes, you are correct. I had it included previously since I was initially creating the shuttle schedule through this widget but changed course thereafter. I will remove this dependency.